src/pages/MainLayout.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <div>
+  <div v-if="browserDetect">
+    <p>You are using an unsupported browser. Please <a
```

Comment:
Can you make it translatable? A significant number of people have no grasp of English language.

browser-update.org is also multi-lingual, so that makes sense to me.

Oh by the way, can you set the link to open in a new window? Don't forget to add `noopener` to avoid XSS attacks: https://mathiasbynens.github.io/rel-noopener/

Reply:
Certainly! Are you referring to adding the [`translate`](https://www.w3.org/International/questions/qa-translate-flag) flag to the `<p>` and `<a>` elements? Or is there something I should do related to Transifex?